modules/ROOT/pages/_partials/release_notes/sync-gateway-3-1-5-release-note.adoc

```diff
@@ -0,0 +1,64 @@
+== 3.1.5 -- April 2024
+
+Version 3.1.5 of Sync Gateway delivers the following features and enhancements:
+
+NOTE: For an overview of the latest features offered in Sync Gateway 3.1, see xref:whatsnew.adoc[New in 3.1].
+
+[WARNING]
+--
+
+We have discovered a critical issue impacting a few customers that utilize OIDC. 
```

Comment:
Should this be clearer that the issue is with `3.1.4`?

Reply:
Can make this clearer yes!